Tests/SourceKitLSPTests/LocalSwiftTests.swift

```diff
@@ -621,6 +621,11 @@ final class LocalSwiftTests: XCTestCase {
       diagnostic = note.params.diagnostics.first!
     })
 
+    guard let diagnostic = diagnostic else {
```

Comment:
An alternative spelling
```suggestion
   let diagnostic = try XCTUnwrap(diagnostic, "...")
```

Reply:
Hmm, I definitely prefer that, thanks!